luarules/gadgets/unit_minesweeper_detection.lua

```diff
@@ -0,0 +1,245 @@
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+
+function gadget:GetInfo()
+	return {
+		name = "Minesweeper Detection",
+		desc = "Shows mines around minesweepers",
+		author = "Hornet",
+		date = "May 25th, 2024",
+		license = "GNU GPL, v2 or later",
+		layer = 0,
+		enabled = true
+	}
+end
+
+if not gadgetHandler:IsSyncedCode() then
+	return false
+end
+
+local spMoveCtrlSetAirMoveTypeData = Spring.MoveCtrl.SetAirMoveTypeData
+
+local minesweeperIDs = {}
+local minesweepers = {}
+local mineIDs = {}
+
+--Spring.Echo('hornet ms loaded')
+
+local teamList = Spring.GetTeamList()
+local teamIDs = {}
+
+local revealedMines = {}=
```

Comment:
Syntax error on this line
```suggestion
local revealedMines = {}
```